PyDSS/defaults/simulation.toml

```diff
@@ -47,6 +47,7 @@
 "Export Compression" = false
 "Export Iteration Order" = "ValuesByPropertyAcrossElements"
 "Export Elements" = true
+"Export Tables" = true
```

Comment:
@AadilLatif This is one item you might feel strongly about.  This option will generate user-friendly tabular data for all element properties (one file per element/element-property).  That means many files.